src/app/components/Badge/index.tsx

```diff
@@ -7,18 +7,21 @@ type Props = {
   small?: boolean;
 };
 
-export default function Badge({ label, color, textColor, small }: Props) {
-  const { t: tComponents } = useTranslation("components", {
-    keyPrefix: "badge",
-  });
+export default function Badge({
+  label = "active",
+  color,
+  textColor,
+  small,
+}: Props) {
+  const { t: tComponents } = useTranslation("components");
 
   return (
     <span
       className={`inline-block leading-none rounded font-medium bg-${color} text-${textColor} ${
         !small ? "p-1.5 text-xs" : "p-1 text-xxxs"
       }`}
     >
-      {tComponents(`label.${label}`)}
+      {tComponents(`badge.label.${label as "active"}`)}
```

Comment:
i18n TS fails to interpret this correctly. Same for some other situations in the PR.